docs/docs/dev-manuals/services/discount.mdx

```diff
@@ -63,41 +63,41 @@ The following types are used in the defined events:
 
 ```tsx
 interface Order {
-  id: string,
-  userId: string,
-  createdAt: Date,
-  orderStatus: OrderStatus,
-  placedAt?: Date,
-  rejectionReason?: RejectionReason,
-  orderItems: OrderItem[],
-  shipmentAddressId: string,
-  invoiceAddressId: string,
-  compensatableOrderAmount: number,
-  paymentInformationId: string
+ id: string;
+ userId: string;
+ createdAt: Date;
+ orderStatus: OrderStatus;
+ placedAt: Date;
```

Comment:
your OrderDTO says `/// Timestamp of Order placement. `None` until Order is placed.`
so I know you only send this when it is placed, but I would say we either leave it here possibly undefined or you fix the docs in the order service to not have any inconsistencies here

Reply:
This can no longer be undefined since we have fixed the time format on serialization, i can fix the docs there